src/main/java/dev/latvian/mods/kubejs/util/MobEffectUtil.java

```diff
@@ -0,0 +1,39 @@
+package dev.latvian.mods.kubejs.util;
+
+import net.minecraft.core.Holder;
+import net.minecraft.world.effect.MobEffect;
+import net.minecraft.world.effect.MobEffectInstance;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+
+public class MobEffectUtil {
```

Comment:
Can `@Info` annotations be put on the class and each method to document them.

Reply:
Yes, I can do that.